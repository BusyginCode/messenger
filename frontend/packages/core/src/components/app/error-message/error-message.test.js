import React from 'react'
import { shallow } from 'enzyme'
import ErrorMessage from './error-message'

describe('App/ErrorMessage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<ErrorMessage error={new Error('404')} />)
    expect(wrapper).toMatchSnapshot()
  })
})
