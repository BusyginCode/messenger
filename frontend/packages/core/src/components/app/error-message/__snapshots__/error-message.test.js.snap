// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App/ErrorMessage should render correctly 1`] = `
<div>
  <h1>
    Something bad happened :(
  </h1>
  <div>
    Error: 404
  </div>
</div>
`;
